### ISSUE TITLE
Executable Spring Boot archives are not supported (Unexpected record signature: 0X622F2123)

### ISSUE THREAD

**Issue by vitalyster** [ayltai/spring-graalvm-native-plugin#43]
**Things to check before reporting a bug**

- [x] I have searched [GraalVM GitHub repository](https://github.com/oracle/graal/issues) and could not find a similar issue 
- [x] I have searched [Spring experimental GraalVM Native project](https://github.com/spring-projects-experimental/spring-graalvm-native/issues) and could not find a similar issue
- [x] I have attached a sample project so that the issue can be reproduced on other computers

**Describe the bug**

build failed with error message above

**To Reproduce**
Add to build.gradle:
```
bootJar {
    launchScript()
}
```

**Expected behavior**

`buildNativeImage` task should build image

**Comment by ayltai** [comment]
The error message means the JAR file generated by `bootJar` is not a valid/standard zip file. Can you share a simple test repository that I can use to reproduce the error locally?

**Comment by vitalyster** [comment]
Just add `launchScript()` to `bootJar` configuration of any Spring Boot application and Spring Boot will repackage jar with executable script, see https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/#packaging-executable-configuring-launch-script

**Comment by ayltai** [comment]
RIght, on the documentation it says "Currently, some tools do not accept this format so you may not always be able to use this technique" so apparently the JAR file generated is not supported by this plugin which uses Java API to extract the contents from the JAR file.

Why do you want to produce an executable JAR while at the end you want a native image?

**Comment by vitalyster** [comment]
@ayltai I just follow guide to enable native image plugin and see this error. If it is too much work to add support for this non-standard jar then guide should mention that option and ask user to remove it

**Comment by ayltai** [comment]
Good point. I wasn't aware of that. Thanks
